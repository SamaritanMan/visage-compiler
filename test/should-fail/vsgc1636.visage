/**
 * Regression test for VSGC-1636 : For now, disallow all blank 'def's
 *
 * @test/fail
 */

def a : Integer;

class Foo {

   def b : String;

   function bar() {
      def c : Foo;
   }
}

