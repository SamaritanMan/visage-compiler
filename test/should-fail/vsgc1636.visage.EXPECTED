vsgc1636.visage:7: The def a must have an initializing expression
def a : Integer;
^vsgc1636.visage:11: The def b must have an initializing expression
   def b : String;
   ^
vsgc1636.visage:14: The def c must have an initializing expression
      def c : Foo;
      ^
3 errors